pkg/kosmosctl/join/join.go

```diff
@@ -80,8 +80,8 @@ func NewCmdJoin(f ctlutil.Factory) *cobra.Command {
 		SilenceUsage:          true,
 		DisableFlagsInUseLine: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			ctlutil.CheckErr(o.Complete(f))
```

Comment:
It is recommended not to change the order here, 'Validate' content can be optimized

Reply:
order of Complete and Validate restored.